frontend/src/app/actions/addon.js

```diff
@@ -3,5 +3,6 @@ import { createActions } from 'redux-actions';
 export default createActions(
   {},
   'setHasAddon', 'setInstalled', 'setClientUuid',
-  'enableExperiment', 'disableExperiment'
+  'enableExperiment', 'disableExperiment',
```

Comment:
Filed https://github.com/mozilla/testpilot/issues/1371 to use constants for these.